src/Symfony/Component/Form/Extension/Validator/ValidatorTypeGuesser.php

```diff
@@ -155,8 +155,8 @@ public function guessTypeForConstraint(Constraint $constraint)
             case 'Symfony\Component\Validator\Constraints\Count':
                 return new TypeGuess('collection', array(), Guess::LOW_CONFIDENCE);
 
-            case 'Symfony\Component\Validator\Constraints\True':
-            case 'Symfony\Component\Validator\Constraints\False':
+            case 'Symfony\Component\Validator\Constraints\IsTrue':
+            case 'Symfony\Component\Validator\Constraints\IsFalse':
```

Comment:
the deprecated names should be kept here (this is the issue when using a ``switch(get_class()``: it does not support inheritance)